app/_layouts/oas/index.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
```

Comment:
Will we have option to add description metadata to this page?

Reply:
yes